src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPluginExtension.kt

```diff
@@ -9,14 +9,14 @@ import org.gradle.api.Project
 open class MavenPublishPluginExtension(project: Project) {
   /**
    * The release repository url this should be published to.
-   * This defaults to the sonatypes url.
+   * This defaults to either the RELEASE_REPOSITORY_URL Gradle property, the system environment variable or the snapshot url.
```

Comment:
Typo? This shouldn't default to the `snapshot` url.

Reply:
Yes. I'll fix it.